client/calendar.scss

```diff
@@ -36,7 +37,12 @@
   width: 100px;
   background-color: #ccc;
```

Comment:
use variables for colors and dimensions like `#ccc` and `100px` this is the advantage of sass. You can declare a variable by doing this:
```
$light-gray: #ccc;


#calendar-id th {
  background-color: $light-gray;
  ...
}
```